easybuild/easyconfigs/p/PyTorch/PyTorch-2.1.0_skip-float16-part-test_batchnorm_nhwc_cpu.patch

```diff
@@ -0,0 +1,35 @@
+Skip a float16 test where the error was slightly above tolerance.  It was
+not obvious how to increase the tolerance.
+
+Author: Jakob Schiotz, Tech. Univ. Denmark
+Email: schiotz@fysik.dtu.dk
+Date: 8. November 2023
+
+diff --git a/test/distributed/elastic/multiprocessing/api_test.py b/test/distributed/elastic/multiprocessing/api_test.py
+index 6240fbe..35990be 100644
+--- a/test/distributed/elastic/multiprocessing/api_test.py
++++ b/test/distributed/elastic/multiprocessing/api_test.py
+@@ -658,7 +658,9 @@ if not (TEST_WITH_DEV_DBG_ASAN or IS_WINDOWS or IS_MACOS):
+ 
+ 
+ # tests incompatible with tsan or asan, the redirect functionality does not work on macos or windows
+-if not (TEST_WITH_DEV_DBG_ASAN or IS_WINDOWS or IS_MACOS or IS_CI):
++# Test disabled for EasyBuild, it fails with python_std.fileno() raising 
++# io.UnsupportedOperation: fileno
++if False:
```

Comment:
This part of the patch doesn't match the description and the rest. Might be worth doing that in a separate test with a description and if possible stacktrace of the error. Actually a stacktrace including the diff exceeding the tolerance would also be good here such that one can find this patch if one encounters the error without this.

Reply:
Oops, I have messed up when I tried to use a local git repo to keep track of the patches I made.  I have separated it into two patches.

As for including stack traces: I assume you mean including the stack trace in the text at the top of the patch.  I have not seen that done before, but it certainly makes a lot of sense, I can see it will make it easier for you or others to evaluate whether to include the patch in future versions.  Stack traces added.

Thank you @Flamefire for your review!

Reply:
Thank you!   
And yes this hasn't been done consistently before. At least in more recent patches I included either the (reduced) stacktrace, i.e. condensed to enough to be discoverable after I noticed the failures are pretty much the same, like e.g.

```
AssertionError: Tensor-likes are not close!
Mismatched elements: 4 / 16 (25.0%)
Greatest absolute difference: 1.0 at index (1, 0) (up to 1e-05 allowed)
Greatest relative difference: 0.3333333432674408 at index (1, 0) (up to 1.3e-06 allowed)
```
Together with the name of the test this is often already enough.   
Alternatively as I usually report each issue in the PyTorch Github repo I link that with a very short description as the issue contains much more complete information already.